corehq/apps/translations/templates/download_translations.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'hqwebapp/two_column.html' %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% requirejs_main "translations/js/app_translations" %}
+{# TODO: Check that #}
+
+{% block page_content %}
+  {% registerurl "paginate_releases" domain '---' %}
+  {# TODO: Check that #}
+  {% if not transifex_details_available %}
+    <p class="text-error">
```

Comment:
UI nitpick: it's more standard to use a div with `alert-danger` for this kind of message, rather than a `text-error` with a `label-danger` inside.